grammars/scss.cson

```diff
@@ -291,27 +291,34 @@
   'at_rule_include':
     'patterns': [
       {
-        'begin': '\\s*((@)include\\b)\\s*'
+        'name': 'meta.at-rule.include.scss'
```

Comment:
I noticed this in the surrounding code, but this ordering actually makes the code much harder to read. Do we really have to do that?

Every [tutorial](http://manual.macromates.com/en/language_grammars.html) I read about creating Textmate bundles puts `name` on the top in their examples. I think this makes more sense: 'name' is just like a title of an article here—it tells you what the pattern is about, you wouldn't put the title of an article in the middle of everything; you often need to use the 'name' somewhere else, putting it in the middle makes it much harder to find, which is really bothersome when you try to do it in the surrounding code.